include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -120,6 +120,12 @@ const OrtApi* Global<T>::api_ = OrtGetApiBase()->GetApi(ORT_API_VERSION);
 /// This returns a reference to the OrtApi interface in use
 inline const OrtApi& GetApi() noexcept { return *Global<void>::api_; }
 
+/// <summary>
+/// This Api returns the version string from the compiled onnxruntime binary
+/// </summary>
+/// <returns>version string major.minor.rev</returns>
+std::string GetVersionString();
```

Comment:
Is the API version or the binary version? Should we rename it to GetOrtBinaryVersionString() or something better?

Reply:
I put the name the same as C API. Also the doc says it all. I can add the word binary to the doc.